### DISCUSSION TITLE
Bitwarden Linux Desktop client does not support FIDO2/WebAuthn with Yubikey

### DISCUSSION THREAD

**Discussion by rajil** [dani-garcia/vaultwarden#2262] [Q&A]
### Subject of the issue
Bitwarden Desktop 1.30 on Linux does not support Webauthn
### Deployment environment

* vaultwarden version:
Version 1.23.1

* Install method:
Docker

* Clients used: 
Desktop
* Reverse proxy and version: 
Nginx

* Other relevant details:

### Steps to reproduce
1. Setup Vault Warden to use 2FA using FIDO2 WebAuthn and Yubikey
2. Use Desktop client to open the vault and was greeted with,

![image](https://user-images.githubusercontent.com/379266/151223831-d08a3b76-ad04-42bc-990c-5429bca7ee16.png)

**Comment by BlackDex** [comment]
Desktop clients are not maintained or provided by Vaultwarden
